src/NzbDrone.Core.Test/Datastore/Migration/188_mediainfo_channel_propertiesFixture.cs

```diff
@@ -0,0 +1,105 @@
+using System;
+using System.Linq;
+using FluentAssertions;
+using NUnit.Framework;
+using NzbDrone.Common.Serializer;
+using NzbDrone.Core.Datastore.Migration;
+using NzbDrone.Core.Test.Framework;
+
+namespace NzbDrone.Core.Test.Datastore.Migration
+{
+    [TestFixture]
+    public class mediainfo_channel_propertiesFixture : MigrationTest<mediainfo_channels>
+    {
+        private void AddMovieFile(mediainfo_channels m, int id)
+        {
+            var movie = new
+            {
+                Id = id,
+                MovieId = id,
+                Quality = new { }.ToJson(),
+                Size = 0,
+                DateAdded = DateTime.UtcNow,
+                RelativePath = "SomeFile.mkv",
+                Languages = new[] { 1, 2 }.ToJson(),
```

Comment:
Im pretty sure Languages and a few other properties here need to be changed for Sonarr..